W-15225608 - Fixing external tester tool and adding regex flavor. (#345)
mulesoft/docs-rpa: rpa-builder/6.5/modules/ROOT/pages/advanced-concepts-using-regular-expressions.adoc

= Using Regular Expressions

Regular Expression or Regex are patterns that provide a means to specify and recognize texts. Some action steps support using Regex in their configuration.

RPA Builder uses C# flavor for Regex and includes the Regex Tester tool, which enables you to xref:advanced-concepts-using-regular-expressions-testing-regular-expression.adoc[test regular expressions].

For specific use cases and examples, see xref:advanced-concepts-using-regular-expressions-examples.adoc[Regex Examples].

> If you need further information and references, following are some links to get you started with Regex:

Reviewer: ```suggestion
If you need to start using Regex, you can get information and references in the following links:

```
